src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="form"
+      method="post"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search"
+        class="icon icon--big"
+      >
+      <label>
+        <input
+          type="text"
+          class="search-bar"
+          name="searchBig"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form
+      action="#"
+      class="form form-small"
+      data-qa="small"
+    >
+      <img
+            src="images/Search.svg"
+            alt="search"
+            class="icon icon-small"
+      >
```

Comment:
please pay attention and check the code formatting
```suggestion
      <img
        src="images/Search.svg"
        alt="search"
        class="icon icon-small"
      >
```